docs/docs/configuration/animations.mdx

```diff
@@ -119,9 +127,8 @@ The default configuration is defined here: <a href="https://github.com/chartjs/C
 | `debug` | `boolean` | `undefined` | Running animation count + FPS display in upper left corner of the chart.
 | `delay` | `number` | `undefined` | Delay before starting the animations.
 | `loop` | `boolean` | `undefined` | If set to `true`, the animations loop endlessly.
-| [[mode]](#animation-mode-configuration) | `object` | [defaults...](#default-modes) | Option overrides for update mode. Core modes: `'active'`, `'hide'`, `'reset'`, `'resize'`, `'show'`. See **Hide and show [mode]** example above.
-| [[property]](#animation-property-configuration) | `object` | `undefined` | Option overrides for a single element `[property]`. These have precedence over `[collection]`. See **Looping tension [property]** example above.
-| [[collection]](#animation-properties-collection-configuration) | `object` | [defaults...](#default-collections) | Option overrides for multiple properties, identified by `properties` array.
+| `modes` | `object` | [defaults...](#default-modes) |  Option overrides for update modes. Core modes: `'active'`, `'hide'`, `'reset'`, `'resize'`, `'show'`. See **Hide and show [mode]** example above.
```

Comment:
I would suggest to rename this option to something clearer. For user not familiar with the `update` argument, it's hard to get what is it and from where it comes.

What do you think of `transitions` as suggested in [this comment](https://github.com/chartjs/Chart.js/issues/8213#issuecomment-749461754)?

Reply:
I think it might be better name for it. Just a lot of changes if doing `mode`->`transition` everywhere.